packages/forma-36-website/gatsby-config.js

```diff
@@ -296,6 +304,15 @@ module.exports = {
   plugins: [
     'gatsby-plugin-react-helmet',
     'gatsby-plugin-svgr',
+    {
+      resolve: `gatsby-source-git`,
+      options: {
+        name: `forma-36-react-components`,
+        remote: `https://github.com/contentful/forma-36.git`,
+        branch: `master`,
```

Comment:
@burakukula Ah I see now, because we're using git as a source

Reply:
@m10l yeah, exactly :)

Reply:
Do we need to do this? Can't we use the code in the branch always using relative paths or something? (so when creating a PR the preview would use the current code of that branch)

Reply:
@gui-santos it's an option as well, but I just thought it's better to rely on GtiHub, where files are verified already. But I am open to change the approach. Maybe it would actually be more comfortable during development. Maybe I would do it on a separate branch? I would like to move files from website to README's first.

Reply:
IMO, it would be better to be able to preview by change made in the repo right way. For example, you fixed the error messages Mike and I commented before but we can't see if they are really fixed since the changes are in this branch and not in master

But I guess we can do it in another branch of course (maybe the website will have some errors idk)